### PR TITLE
test(react-color-picker): added vr tests

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
Added VR tests for the ColorPicker

### CODE REVIEW

File: apps/vr-tests-react-components/src/stories/ColorPicker/ColorPicker.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export default {
+  title: 'ColorPicker Converged',
+  decorators: [story => withStoryWrightSteps({ story, steps })],
+} satisfies Meta<typeof ColorPicker>;
+
+export const Default = () => (
+  <>
+    <SampleColorPicker color={{ h: 109, s: 1, v: 0.91 }} />
+  </>
+);
```
Comment (mainframev) [inline apps/vr-tests-react-components/src/stories/ColorPicker/ColorPicker.stories.tsx:17]: No need for fragment here :) 

```suggestion
export const Default = () => <SampleColorPicker color={{ h: 109, s: 1, v: 0.91 }} />;
```

File: apps/vr-tests-react-components/src/stories/ColorPicker/ColorPicker.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Default.storyName = 'default';
```
Comment (mainframev) [inline apps/vr-tests-react-components/src/stories/ColorPicker/ColorPicker.stories.tsx:18]: why we need the storyName for default and shape? 🤔

File: apps/vr-tests-react-components/src/stories/ColorPicker/utils.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+export const steps = new Steps()
+  .snapshot('default', { cropTo: '.testWrapper' })
+  .hover('.breadcrumb-sample')
+  .snapshot('hover', { cropTo: '.testWrapper' })
+  .mouseDown('.breadcrumb-sample')
+  .snapshot('pressed', { cropTo: '.testWrapper' })
+  .focus('.breadcrumb-sample')
+  .snapshot('focused', { cropTo: '.testWrapper' })
+  .end();
```
Comment (mainframev) [inline apps/vr-tests-react-components/src/stories/ColorPicker/utils.tsx:19]: I think it was copied and was not adjusted to ColorPicker 🙂

**Review by mainframev — CHANGES_REQUESTED** [review]

File: apps/vr-tests-react-components/src/stories/ColorPicker/ColorPicker.stories.tsx (hunk 2 above)
Comment (ValentinaKozlova) [reply apps/vr-tests-react-components/src/stories/ColorPicker/ColorPicker.stories.tsx:14]: Followed code convention. But I see now that everybody make in differently

**Review by mainframev — APPROVED** [review]

**Pull request merged**